### PR TITLE
[ADD] hr_employee_category_parent

### PR DESCRIPTION
**Pull request by hbrunn**
HR category hierarchy
=====================

This module was written to restore the employee category hierarchy from v8 and before.

Usage
=====

To use this module, you need to:

- go to some employee category and assign a parent

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and technical review :+1:

File: hr_employee_category_parent/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=====================
+HR category hierarchy
```
Comment (MiquelRForgeFlow) [inline hr_employee_category_parent/README.rst]: HR Employee Category Parent

File: hr_employee_category_parent/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Employee category hierarchy",
```
Comment (MiquelRForgeFlow) [inline hr_employee_category_parent/__manifest__.py]: HR Employee Category Parent

**Comment by feketemihai** [comment]
@hbrunn Can you update the name of the module as @mreficent suggested, otherwise LGTM...

**Comment by hbrunn** [comment]
done

**Review by feketemihai — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**